docs/cel2/operators/migrate-node.md

```diff
@@ -112,7 +112,39 @@ Alternatively, you can run the check-db script on its own by checking out the la
 
 ## Migrating a Celo archive node
```

Comment:
I find the title ambiguous. To me, it's amalgamating
- running the migration on an archive node (but then it just says not to do it)
- running an L2 archive node (this is explained)

Not sure where, but I'd make it explicit that for archive node, it's required to start with an L1 (non archive) full node and an L1 archive node and that the final setup is an L2 archive "node" and an L1 archive node.

Maybe a diagram?